src/generate/apiGroup.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import _ from "lodash";
+
+import { Api } from "./api";
+import Name from "./name";
+
+/**
+ * A group of API objects. Common transformations of the group name are cached
+ * for reference in templates and file paths.
+ */
+export class ApiGroup {
+  apis: Api[];
+  name: Name;
+
+  constructor(name = "", apis?: Api[]) {
+    this.name = new Name(name);
+    this.apis = apis;
```

Comment:
```suggestion
    this.apis = apis || [];
```
The constructor parameter is optional, but the class definition requires it to be set. (Alternative to this is to update the parameter or class definition.)

Reply:
updated parameter